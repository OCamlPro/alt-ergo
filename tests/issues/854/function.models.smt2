(set-logic ALL)
(set-info :smt-lib-version 2.6)
(set-option :produce-models true)
(declare-sort intref 0)
(declare-fun intrefqtmk (Int) intref)
(declare-fun a () Int)
(declare-fun f (Int) Int)
(define-fun aqtunused ((x Int)) intref (intrefqtmk (f a)))
(assert (= (aqtunused 0) (aqtunused 1)))
(declare-fun a1 () Int)
(assert (not (and (<= 5 a1) (<= a1 15))))
(check-sat)
(get-model)
