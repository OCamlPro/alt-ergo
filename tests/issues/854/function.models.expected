
unknown
(
  (define-fun intrefqtmk ((_arg_0 Int)) intref (as @a0 intref))
  (define-fun a () Int 0)
  (define-fun f ((_arg_0 Int)) Int 0)
  (define-fun a1 () Int 0)
)
